src/hooks/useUploadsManager/useUploadsManager.tsx

```diff
@@ -54,7 +54,16 @@ export const UploadManagerProvider: React.FC = ({ children }) => {
     },
     { skip: !uploadsState.length }
   )
-  const pendingNotificationsCounts = useRef({ uploading: 0, uploaded: 0 })
+
+  const pendingUploadingNotificationsCounts = useRef(0)
+  type AssetParentObjectID = string
+  const assetsNotificationsCount = useRef<{
+    uploading: AssetParentObjectID[]
+    uploaded: AssetParentObjectID[]
+  }>({
+    uploading: [],
+    uploaded: [],
+  })
```

Comment:
Wouldn't it be easier to just use an objects here instead? So something like
```
{
  uploading: {
    '12': 2
  }
}
```

Reply:
Maybe it would, but now I added type of parent object to it so I'm not so sure about how would we store parent object type as well with that solution

Reply:
Could we just use something like `${parentType}-${parentId}` as a key in that object? So for example `channel-12`